### PR TITLE
feat: save shipping address to my account

### PR DESCRIPTION
**Pull request by kibo-sushant**

### CODE REVIEW

File: components/KiboShipping.vue [hunk 1]
```diff
@@ -6,13 +6,44 @@
...
+          @validateForm="validateShippingDetails"
+          @toggleAddressFormVisibility="toggleAddressFormVisibility"
+        />
+      </SfLoader>
+    </div>
+
+    <div v-if="isAddressFormOpen && isUserLoggedIn">
+      <SfCheckbox
+        name="shipping"
+        label="Save shipping address"
```
Comment (kibo-susantasamanta) [inline components/KiboShipping.vue]: I think internationalization should be used here.

File: components/KiboShipping.vue [hunk 2]
```diff
@@ -6,13 +6,44 @@
...
+        error-message=""
+        valid
+        :disabled="false"
+        :selected="isSaveShippingAddresChecked"
+        @change="changeAddressSave"
+      />
+
+      <SfCheckbox
+        name="shipping"
+        label="Make this my default address"
```
Comment (kibo-susantasamanta) [inline components/KiboShipping.vue]: Same, internationalization should be used here.

File: components/KiboShipping.vue [hunk 3]
```diff
@@ -49,11 +82,36 @@ export default {
...
+      type: Boolean,
+      default: false,
+    },
   },
   setup(props, context) {
     const isReadonly = true
     const isValidShippingDetails = ref(false)
     const shippingAddress = ref({ ...props.value })
+    const isAddressFormOpen = ref(false)
+    const isSaveShippingAddresChecked = ref(false)
```
Comment (kibo-susantasamanta) [inline components/KiboShipping.vue]: Is this intentionally used in few places  'Addres' and for others 'Address' ?

File: components/KiboShipping.vue (hunk 1 above)
Comment (kibo-sushant) [reply components/KiboShipping.vue]: Resolved

File: components/KiboShipping.vue (hunk 2 above)
Comment (kibo-sushant) [reply components/KiboShipping.vue]: Resolved

File: components/KiboShipping.vue (hunk 3 above)
Comment (kibo-sushant) [reply components/KiboShipping.vue]: Resolved

**Pull request closed without merging**